examples/webpack.config.local.js

```diff
@@ -50,9 +49,7 @@ function makeLocalDevConfig(EXAMPLE_DIR = LIB_DIR) {
           enforce: 'pre'
         }
       ]
-    },
-    // Optional: Enables reading mapbox token from environment variable
-    plugins: [new webpack.EnvironmentPlugin(['MapboxAccessToken'])]
```

Comment:
This causes issues when example's webpack version is different from root. All examples with base map already specifies this plugin in their own webpack config.

Reply:
Should we move all examples to webpack 4? I am doing that in luma.gl right now.